packages/@webex/plugin-messages/src/messages.js

```diff
@@ -160,6 +160,83 @@ const Messages = WebexPlugin.extend({
 
     return this.request(options).then((res) => res.body);
   },
+  /**
+   * Put an updated message and/or media content into a room instead of existing message.
+   * @instance
+   * @memberof Messages
+   * @param {MessageObject} message
+   * @param {MessageObject} altMessage
+   * @returns {Promise<MessageObject>}
+   * @example
+   * webex.rooms.create({title: 'Create Message Example'})
+   *   .then(function(room) {
+   *     return webex.messages.create({
+   *       text: 'Howdy!',
+   *       roomId: room.id
+   *     });
+   *   })
+   *   .then(function(m) {
+   *     message = m;
+   *     return webex.messages.update(message,{markdown:`**What up**`});
+   *   })
+   *   .then(function(m) {
+   *     message = m;
+   *     return webex.messages.update(message.id,{roomId:message.roomId,text:'Howdy!'});
+   *   })
+   *   .then(function(message) {
+   *     var assert = require('assert');
+   *     assert(message.id);
+   *     assert(message.personId);
+   *     assert(message.personEmail);
+   *     assert(message.roomId);
+   *     assert(message.created);
+   *     return 'success';
+   *   });
+   *   // => success
+   */
+  update(message, altMessage) {
+    const id = message.id || message;
```

Comment:
@arun3528 Do we need these changes in beta and next?
https://github.com/webex/webex-js-sdk/pull/2708

Reply:
yes i remember someone added it , its been a while , why what happened ?

Reply:
@Shreyas281299 - Can we resolve this?

Reply:
Yes. Resolving